src/ResourceManager/ContainerInstance/ChangeLog.md

```diff
@@ -19,6 +19,12 @@
 -->
 ## Current Release
 
+## Version 0.1.1
```

Comment:
@yolo3301 please remove the `## Version 0.1.1` header and move the below content to be under `## Current Release`